tests/unit/utils/config-utils.spec.js

```diff
@@ -0,0 +1,57 @@
+describe('isProd', () => {
```

Comment:
I deprecated all the current unit tests since they were't being used as far as I can tell in any of your tests. We can put them back of course but I wanted to check with you first.

That said, I updated the jest implementation so that I could add some tests for my `isProd` utility. You can test it with `npm run test:unit`.